src/lib/components/comparisonTable/components/TableButton/style.module.scss

```diff
@@ -2,17 +2,26 @@
 
 .button {
   background-color: transparent;
-  border-bottom: 2px dashed $ds-grey-600;
   color: $ds-grey-700;
   cursor: pointer;
+  margin: 2px 0;
   padding: 2px;
-  transition: 0.3s ease;
-  transition-property: color, border-color;
+  text-align: left;
```

Comment:
Hardcoding a left alignment for text is not a good idea as it only really works if the button is used as the table row header. If it's used to turn some other text into a clickable unit, it will lead to inconsistencies as the default is centered text:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/13664983/218419002-adab157d-8229-450f-8609-103f3349151e.png">

Reply:
Thanks for this catch! Completely missed the data table/content with multi line. Kept center as default and aligning left on table row headers only, as it should.